Makefile

```diff
@@ -74,10 +76,18 @@ endif
 	@printf "The engine has been cleaned.\n"
 
 version: variables
+ifneq ($(OVERWRITE_VERSION),"True")
+	@printf "Setting mod version to $(MOD_VERSION)\n"
```

Comment:
Alternatively "Mod version $(MOD_VERSION) will remain unchanged."

Reply:
Yes, please change this.

Reply:
Done.